README.md

```diff
@@ -47,65 +41,35 @@ The Safe Network is a decentralised and autonomous data network, built atop kade
 - [Build Info](https://github.com/maidsafe/sn_build_info/blob/master/README.md) Small helper used to get the build/commit versioning info for debug purposes.
 
 
-## Running the network
-
-You'll need to set the `SAFE_PEERS` env variable to the multiaddress of a node when you set up a testnet.
-You can do this automatically at network startup using the following command (if you have ripgrep installed)
+## Run your MaidSafe Local Test Network in 4 easy Steps
+Before you start, make sure to have **[Rust installed](https://www.rust-lang.org/learn/get-started)** and the latest version with `rustup update`
+<br>
+1. Create your Local Test Network : <br>
+```bash
+Cargo run --bin testnet --features="local-discovery" -- --build-node --build-faucet --interval 100 --clean
```

Comment:
Sorry for the pedantic point, but probably better to have the `cargo` commands in lower case?

Reply:
I changed it to upper case to workaround an issue I was having but I have learned since I am having issues with my editor. I can change this back. (in picture two arrows instead of one, and the single around the command would not work = editor issue
![Screenshot 2024-01-10 123333](https://github.com/maidsafe/safe_network/assets/11191438/e5b54f6d-a92f-47e0-b74b-609eaecb7d3c)
)